roles/kubernetes/master/tasks/encrypt-at-rest.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Write secrets for encrypting secret data at rest
+  template:
+    src: secrets_encryption.yaml.j2
+    dest: "{{ kube_config_dir }}/secrets_encryption.yaml"
+    mode: 0640
+  tags:
+    - kube-apiserver
```

Comment:
missing condition:
`when: kube_encrypt_secret_data`

Reply:
The whole file is not run if `kube_encrypt_secret_data` is not set. See:

```
- import_tasks: encrypt-at-rest.yml
  when: encrypt_secret_data
```